### DISCUSSION TITLE
Does process_js Options struct support mangle with minify?

### DISCUSSION THREAD

**Discussion by eighty4** [swc-project/swc#9187] [General]
I couldn't find a way to achieve the Compiler.minify's minify and mangle params with process_js.

Would the correct approach be to use TransformOutput.code to create an `Arc<SourceFile>` and call `Compiler.minify` after `process_js`?

**Comment by kdy1** [comment]
https://rustdoc.swc.rs/swc/config/struct.JscConfig.html#structfield.minify

**Comment by eighty4** [comment]
Ah so the Config.minify bool will enable or disable minifying while the JscConfig minify customizes minifying. Definitely missed that.

Thanks.
